cookbooks/apt/recipes/default.rb

```diff
@@ -158,7 +158,7 @@
   uri "https://releases.wikimedia.org/debian"
   distribution "jessie-mediawiki"
   components ["main"]
-  key "90E9F83F22250DD7"
+  key "AF380A3036A03444"
```

Comment:
As already discussed this is already changed on master if you rebase.

Reply:
I merged in your upstream changes, thanks.